monitors/test/ContractMonitor.js

```diff
@@ -319,15 +326,11 @@ contract("ContractMonitor.js", function(accounts) {
       from: disputer
     });
 
-    // Advance time and settle
-    let timeAfterLiquidationLiveness = liquidationTime + 10;
-    await mockOracle.setCurrentTime(timeAfterLiquidationLiveness.toString());
```

Comment:
It is not neccessary to move time forward when before the first liquidation in this test.